documentation/chatmeleon-swagger.yaml

```diff
@@ -181,13 +198,31 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Message"
+                type: object
+                properties:
+                  messageId:
+                    type: string
+                    pattern: "^[0-9a-f]{24}$"
```

Comment:
Add createdAt field to this response, to substitute the individual GET message route earlier